torch/testing/_internal/distributed/_tensor/common_dtensor.py

```diff
@@ -81,35 +76,6 @@ def skip_unless_torch_gpu(method: T) -> T:
     return cast(T, skip_if_lt_x_gpu(NUM_DEVICES)(method))
 
 
-@dataclass
-class RedistributeProfile:
-    num_calls: int
-
-
-@contextmanager
-def redistribute_profiler() -> Generator[RedistributeProfile, None, None]:
```

Comment:
looks like this is an orthogonal refactor, could stamp it if you split PR

Reply:
split it into a PR above :)